cmd/otelcontribcol/go.mod

```diff
@@ -1009,3 +1009,5 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/connector/coun
 replace github.com/openshift/api v3.9.0+incompatible => github.com/openshift/api v0.0.0-20180801171038-322a19404e37
 
 replace github.com/outcaste-io/ristretto v0.2.0 => github.com/outcaste-io/ristretto v0.2.1
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/influx => ../../internal/influx
```

Comment:
Does this need to go in the `builder-config.yaml` file so that it survives code generation?

Reply:
fixed